Obsidian/Util/Extensions/Extensions.cs

```diff
@@ -17,6 +17,36 @@ namespace Obsidian.Util.Extensions
     public static class Extensions
     {
         public static readonly Regex pattern = new Regex(@"[A-Z]{2,}(?=[A-Z][a-z]+[0-9]*|\b)|[A-Z]?[a-z]+[0-9]*|[A-Z]|[0-9]+");
+
+        public static (int value, bool add) GetDifference(this short clickedSlot, int size)
```

Comment:
This function is complex and left undocumented, suggesting to add a XML doc, saying what this function is for (in this case Inventory Slots)